### PR TITLE
[com_languages] Solve PHP warnings if language metafile does not exist

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for new Issue.
#### Summary of Changes

This PR is to remove PHP warnings when a language XML meta file is not found.
##### Before patch

image
##### After patch

image
#### Testing Instructions
1. Use 3.5.0 or latest staging
2. Install a second language (ex: fr-FR)
3. Go to `/language` folder and rename the `fr-FR` directory to anything you like.
4. In Joomla admin, go to `Extensions -> Language(s) -> Installed" you will get the PHP warning.
5. Apply patch.
6. Repeat step 4, you will get a joomla warning message.
   (you can now rename the folder back to the is original name)

Note: the same check if the xml file exists is made in `JInstaller::parseXMLInstallFile()`

### CODE REVIEW

File: libraries/cms/application/helper.php [hunk 1]
```diff
@@ -231,7 +231,7 @@ public static function parseXMLLangMetaFile($path)
 		JLog::add('JApplicationHelper::parseXMLLangMetaFile is deprecated. Use JInstaller::parseXMLInstallFile instead.', JLog::WARNING, 'deprecated');
 
 		// Read the file to see if it's a valid component XML file
-		$xml = simplexml_load_file($path);
+		$xml = @simplexml_load_file($path);
```
Comment (mbabker) [inline libraries/cms/application/helper.php]: Codestyle rules generally discourage the use of `@` to suppress errors.  Personally, I'd suggest a `file_exists($path)` check before running the `simplexml_load_file()` function.  I'd also patch `JInstaller::parseXMLInstallFile()` with the same change.

Comment (andrepereiradasilva) [inline libraries/cms/application/helper.php]: done and done

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 46b472ff49ba3b91be8511fce187e4a022e0243b<br /><br />Able to reproduce then #9730 works as decribed - Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9730">issues.joomla.org/joomla-cms/9730</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 46b472ff49ba3b91be8511fce187e4a022e0243b<br /><br />Tested site and admin languages<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9730">issues.joomla.org/joomla-cms/9730</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9730">issues.joomla.org/joomla-cms/9730</a>.</sub>

**Pull request merged**